debian/copyright

```diff
@@ -0,0 +1,36 @@
+Format: http://dep.debian.net/deps/dep5
+Upstream-Name: qpm
+Source: <url://example.com>
+
```

Comment:
I think it would be good to add the license/copyright for the full package here as well. So the following two lines:

```
Copyright: 2015 Cutehacks AS
License: Artistic 2.0
```

Also, there are several MIT (I think) dependencies in the src directory, do those have to be included as separate "Files:" sections? It's up to if you want to add them, I don't know enough about Debian packages to know if it's common or not :)